sky/templates/kubernetes-ray.yml.j2

```diff
@@ -616,7 +651,49 @@ available_node_types:
               {{k8s_resource_key}}: {{accelerator_count}}
               {% endif %}
             {% endif %}
-            
+      
+      {% if high_availability %}
+      pvc_spec:
+        apiVersion: v1
+        kind: PersistentVolumeClaim
+        metadata:
+          name: {{cluster_name_on_cloud}}-{{k8s_high_availability_deployment_volume_mount_name}}
+          namespace: {{k8s_namespace}}
+        spec:
```

Comment:
Shall we make [`storageclass`](https://kubernetes.io/docs/concepts/storage/storage-classes/) customizable?

Reply:
You're right, it currently uses the default StorageClass. Making it customizable via a new config setting (e.g., `kubernetes.high_availability.storage_class_name`) seems like a good idea for flexibility. What do you think?